docs/pages/connect-your-client/introduction.mdx

```diff
@@ -0,0 +1,330 @@
+
+```code
+$ tsh login --proxy=teleport.example.com --user=alice@example.com --auth=your-idp-connector
+Enter password for Teleport user alice:
+Tap any security key
+> Profile URL:        https://teleport.example.com:443
+  Logged in as:       alice
+  Cluster:            example.com
+  Roles:              access
+  Logins:             root, ubuntu, ec2-user
+  Kubernetes:         enabled
+  Valid until:        2022-11-01 22:37:05 -0500 CDT [valid for 12h0m0s]
+  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty, private-key-policy
+```
+
+</TabItem>
+</Tabs>
+
+</ScopedBlock>
+
+<ScopedBlock scope={["cloud"]}>
+
+```code
+$ tsh login --proxy=mytenant.teleport.sh --user=alice
+Enter password for Teleport user alice:
+Tap any security key
+> Profile URL:        https://teleport.example.com:443
+  Logged in as:       alice
+  Cluster:            mytennant.teleport.sh
+  Roles:              access
+  Logins:             root, ubuntu, ec2-user
+  Kubernetes:         enabled
+  Valid until:        2022-11-01 22:37:05 -0500 CDT [valid for 12h0m0s]
+  Extensions:         permit-agent-forwarding, permit-port-forwarding, permit-pty, private-key-policy
+```
+
+Depending on how Teleport was configured for your network, you may need
+additional flags, such as `--auth=a_saml_connector`. Your administrator should
+provide the details required for your particular use case.
+
+</ScopedBlock>
+
+### Teleport Connect
+
+The Teleport Connect app provides all the same access to resources as `tsh` in
+a friendly graphic user interface. After [downloading](https://goteleport.com/download/)
+and installing Teleport Connect, you can log in and initiate sessions for
+Server and Database Access within a single window.
+
+1. Click **CONNECT** to connect to the Teleport cluster:
+
+   ![A new instance of Teleport Connect](../../img/teleport-connect-init.png)
+
+2. Provide the address of your Teleport Cluster (e.g.
+<ScopedBlock scope={["oss", "enterprise"]}>`https://teleport.example.com`</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>`https://mytennant.teleport.sh`</ScopedBlock>
+) and click **NEXT**.
+
+3. Teleport Connect will ask you for your username, password, and MFA. If
+Teleport is integrated with an external Identity Provider (**IdP**), it may
+instead prompt you to authenticate in a browser window with that service.
+
+4. Once authenticated, you can browse and connect to all the resources your
+user is permitted to access:
+
+   ![An example of Teleport Connect, populated with instances](../../img/use-teleport/connect-ui-overview.png)
+
+### Web UI
+
+Teleport provides a web interface for users to interact with. This is usually
+found at the same URL used to connect to Teleport with (e.g.
+<ScopedBlock scope={["oss", "enterprise"]}>`https://teleport.example.com`</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>`https://mytenant.teleport.sh`</ScopedBlock>),
+but can be adjusted by Teleport Administrators. The Web UI provides similar
```

Comment:
```suggestion
but you should confirm the Web UI URL with the team that manages your Teleport deployment. The Web UI provides similar
```
since we don't really have a concept of a Teleport Administrator in the docs